src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -1875,6 +1875,23 @@ def test_vmss_create_and_modify(self):
         self.cmd('vmss delete --resource-group {rg} --name {vmss}')
         self.cmd('vmss list --resource-group {rg}', checks=self.is_empty())
 
+    @ResourceGroupPreparer(name_prefix='cli_test_vmss_scale_in_policy_')
+    def test_vmss_scale_in_policy(self, resource_group):
+        self.kwargs.update({
+            'vmss': 'vmss1'
+        })
+        self.cmd('vmss create -g {rg} -n {vmss} --image centos --scale-in-policy NewestVM', checks=[
+            self.check('vmss.scaleInPolicy.rules[0]', 'NewestVM')
+        ])
+        self.cmd('vmss update -g {rg} -n {vmss} --set scaleInPolicy.rules[0]=OldestVM', checks=[
+            self.check('scaleInPolicy.rules[0]', 'OldestVM')
+        ])
+        self.cmd('vmss update -g {rg} -n {vmss} --set automaticRepairsPolicy.enabled=false automaticRepairsPolicy.gracePeriod=PT5M automaticRepairsPolicy.maxInstanceRepairsPercent=100', checks=[
+            self.check('automaticRepairsPolicy.enabled', False),
+            self.check('automaticRepairsPolicy.gracePeriod', 'PT5M'),
+            self.check('automaticRepairsPolicy.maxInstanceRepairsPercent', 100)
+        ])
```

Comment:
This test is unrelated to this PR. I prefer removing it, moving to another PR or clarifying it in PR's description.

Reply:
Removed